### PR TITLE
Add asunshinehill sandiego recommendations

### LINKED ISSUE #12: Add asunshinehill San Diego file
- [x] Create a **branch**
- [x] Add file/_commits_
- [x] Open a PR
- [x] Have a discussion about the PR (collaborate)
- [ ] Make edits
- [ ] Get :shipit: approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by asunshinehill**
closes #12 

Hey, @jero5765, does this look like a good list of recommendations? 

cc: @beardofedu @hollenberry @crichID @codergrl

### CODE REVIEW

File: asunshinehill-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#San Diego, CA
+
+###Places to Eat
+- Hanna's Gourmet
+- El Zarape
+
+###Places to Go
```
Comment (jero5765) [inline asunshinehill-sandiego.md:7]: do you :surfer: ?

File: asunshinehill-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- Balboa Park
+- Beach
```
Comment (ghost) [inline asunshinehill-sandiego.md]: Any preferred beach?

**Comment by jero5765** [comment]
Those places look great!  Do you recommend any breweries in the area?

**Comment by asunshinehill** [comment]
@jero5765, added beach.  Does this look good to :shipit:?

**Pull request merged**